### PR TITLE
prioritize sheet creation (#353)

### PR DESCRIPTION
**Pull request by rawxfish**

### CODE REVIEW

File: google_api_lib/utils.py [hunk 1]
```diff
@@ -7,6 +8,12 @@
 from celery import Task
 
 
+class Priority(Enum):
```
Comment (rgossiaux) [inline google_api_lib/utils.py:11]: Should we move this somewhere else? Admittedly not really sure where.

**Review by rgossiaux — COMMENTED** [review]
Do we need to add `queue_order_strategy`? https://docs.celeryproject.org/en/stable/userguide/routing.html#redis-message-priorities

File: google_api_lib/utils.py (hunk 1 above)
Comment (rawxfish) [reply google_api_lib/utils.py]: moved to settings.py

**Comment by rawxfish** [comment]
> Do we need to add `queue_order_strategy`? https://docs.celeryproject.org/en/stable/userguide/routing.html#redis-message-priorities

cry

File: smallboard/settings.py [hunk 2]
```diff
@@ -267,13 +267,23 @@
 
 
 # Celery settings
+from enum import Enum
+
+
+class Priority(Enum):
```
Comment (rgossiaux) [inline smallboard/settings.py:273]: Nit: maybe something like `TaskPriority`

**Review by rgossiaux — APPROVED** [review]
Ty!

**Pull request merged**